hypervisor/boot/sbl/multiboot.c

```diff
@@ -54,7 +54,7 @@ static void parse_other_modules(struct acrn_vm *vm, const struct multiboot_modul
 			static int32_t copy_once = 1;
 
 			start = end + 1; /*it is fw name for boot args */
-			snprintf(dyn_bootargs, 100U, " %s=0x%x@0x%x ", start, mod_size, mod_addr);
+			snprintf(dyn_bootargs, 100U, " %s=0x%x@0x%p ", start, mod_size, mod_addr);
```

Comment:
As our implementation of `snprintf` is [here](https://github.com/projectacrn/acrn-hypervisor/blob/948d58fb9c0b30bb4cab72c071e86cd3d0cce01c/hypervisor/lib/sprintf.c#L415), I don't think we support `%p` for now.

Reply:
yes, it is not supported, we need to document the supported formation more clearly...